src/SmartComponents/HybridInventoryTabs/helpers.test.js

```diff
@@ -121,6 +128,25 @@ describe('getEntities', () => {
     ]);
   });
 
+  test('uses group info from edge API when enforce_edge_groups set to false and isInventoryGroupsEnabled flag is false', async () => {
+    useFeatureFlag.mockReturnValue(false);
+    const { result } = renderHook(() =>
+      useGetEntities(handleRefreshMock, undefined, { rule_id: 'test-rule' })
+    );
+
+    let fetchedResult;
+    await act(async () => {
+      fetchedResult = await result.current(...fetchArguments);
+    });
+
+    expect(fetchedResult.results[0].groups).toEqual([
+      { id: 'edge.group.id-1', name: 'edge.group-1' },
+    ]);
+    expect(fetchedResult.results[1].groups).toEqual([
+      { id: 'edge.group.id-2', name: 'edge.group-2' },
+    ]);
```

Comment:
Why groups are returned if both groups-related flags are false?

Reply:
Because of the same requirement ^.